fusion-plugin-react-router/__tests__/plugin.js

```diff
@@ -213,7 +232,7 @@ test('Custom Provider', async () => {
 
   const app = getApp(element);
   const UniversalEvents = getMockEvents({
-    title: '/',
+    title: 'no-matching-route',
```

Comment:
this was actually throwing an error, but the `expect` was async so it was erroring after the tests passed and completed.